src/qdrant_client/points.rs

```diff
@@ -48,4 +53,319 @@ impl Qdrant {
         })
         .await
     }
+
+    pub async fn search_batch_points(
+        &self,
+        request: impl Into<SearchBatchPoints>,
+    ) -> Result<SearchBatchResponse> {
+        let request = &request.into();
+
+        self.with_points_client(|mut points_api| async move {
+            let result = points_api.search_batch(request.clone()).await?;
+            Ok(result.into_inner())
+        })
+        .await
+    }
+
+    pub async fn search_groups(
+        &self,
+        request: impl Into<SearchPointGroups>,
+    ) -> Result<SearchGroupsResponse> {
+        let request = &request.into();
+
+        self.with_points_client(|mut points_api| async move {
+            let result = points_api.search_groups(request.clone()).await?;
+            Ok(result.into_inner())
+        })
+        .await
+    }
+
+    pub async fn batch_updates(
```

Comment:
Do we also want to have blocking functions in the new API?

Something like this:
```rust
pub async fn batch_updates_blocking(
    &self,
    request: impl Into<UpdateBatchPoints>,
) -> Result<UpdateBatchResponse> {
    let mut request = request.into();
    request.wait = Some(true);
    self.batch_updates(request).await
}
```

Otherwise I'll add comments to all functions that can block but don't do by default.

Reply:
no, blocking/non-blocking separation was one of the reasons why we wanted refactoring ;)